src/Card.js

```diff
@@ -7,14 +7,20 @@ import './App.css';
 export class Card extends React.Component {
   handleDelete = (event) => {
     this.props.onDeleteCard({...this.props});
+
+    event.stopPropagation();
```

Comment:
why do you need it?

Reply:
https://learn.javascript.ru/bubbling-and-capturing

otherwise onClick will work extra